style.css

```diff
@@ -0,0 +1,585 @@
+  color: #ff5438;
+  font-size: 1rem;
+  font-weight: 700;
+  line-height: 18px;
+  letter-spacing: -0.143px;
+  text-align: start;
+}
+
+.materials-list {
+  display: flex;
+  flex-direction: column;
+  gap: 8px;
+  margin-top: 20px;
+}
+
+.cta {
+  display: flex;
+  padding: 20px 50px;
+  margin-top: 5%;
+  background-color: #ff4f38;
+  color: #fff;
+  font-size: 16px;
+  line-height: 28px;
+  letter-spacing: -0.16px;
+  border: none;
+  cursor: pointer;
+}
+
+#socials {
+  display: flex;
+}
+
+#footer-title {
+  background-image: url('./assets/Mobile/hero/wallpaperflare.com_wallpaper.jpg');
+  background-size: cover;
+  /* stylelint-disable */
+  background-clip: text;
+  -webkit-background-clip: text;
+  /* stylelint-enable */
+  color: transparent;
+  margin-right: auto;
+}
+
+.product {
+  display: grid;
+  grid-template-columns: 1fr 2fr;
+  grid-template-rows: 0fr;
+  column-gap: 14px;
+  height: 75%;
+}
+
+.product-img-wrapper {
+  width: 100%;
+  height: 80%;
+  overflow: hidden;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  transition: all;
+  transition-duration: 3s;
+}
+
+.product-img-wrapper:hover {
+  height: 90%;
+}
+
+.product-img {
+  max-width: 100%;
+}
+
+.product-content {
+  display: flex;
+  flex-direction: column;
+  padding: 15px 0 0;
+  padding: 17px;
+  transition: all;
+  transition-duration: 3s;
+  margin-bottom: auto;
+}
+
+.product-content:hover {
+  padding: 5px 0 0;
+  padding: 5px;
+}
+
+#products-list {
+  display: flex;
+  flex-direction: column;
+  gap: 18px;
+}
+
+#products-section-title {
+  color: #000;
+  font-size: 20px;
+  font-weight: 400;
+  line-height: 28px;
+  letter-spacing: -1px;
+}
+
+.product-content > :first-child {
+  color: #000;
+  font-size: 15px;
+  font-weight: 700;
+  line-height: normal;
+  letter-spacing: -0.3px;
+}
+
+.product-content > hr {
+  width: 20px;
+  margin: 12px 0 6px;
+  background: #d9d9d9;
+  border: 1px solid #d9d9d9;
+}
+
+#products-wrapper {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+#products-section-title + hr {
+  margin: 4px 0 30px;
+}
+
+#products-section {
+  padding: 15px 12px 32px;
+}
+
+#products-section > .cta.more {
+  display: flex;
+  background-color: transparent;
+  border-radius: 4px;
+  border: 1px solid #dcdcdc;
+  box-shadow: 2px 2px 4px 0 rgb(0 0 0 / 25%);
+  color: #000;
+  font-size: 11px;
+  font-weight: 700;
+  line-height: 28px;
+  letter-spacing: -0.55px;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  padding: 12px 0;
+  gap: 10px;
+}
+
+#partners-list {
+  display: flex;
+  gap: 50px;
+  align-items: center;
+}
+
+#partners {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 35px 120px;
+  background-color: #414246;
+}
+
+#partners h1 {
+  color: #8f8f91;
+}
+
+#partners hr {
+  margin-bottom: 50px;
+}
+
+#footer-section {
+  display: flex;
+  align-items: center;
+  padding: 5% 17%;
+  gap: 65px;
+  max-width: 90%;
+}
```

Comment:
- [ ] Kindly make sure the footer section matches the Behance design, as it should include the logo aligned to the left with some description about the event to the right instead of social media links.
- Behance design:
  ![image](https://github.com/Estete9/IliArtes/assets/104968236/4f07f847-e656-482d-a188-c6960e2edfbd)
- Your design:
  ![image](https://github.com/Estete9/IliArtes/assets/104968236/337d8535-6854-44ff-917c-b712587b3255)